tests/actions.go

```diff
@@ -349,10 +353,44 @@ func (oa *operatorActions) StopInsertDataTo(info *TidbClusterInfo) error {
 	return nil
 }
 
-func (oa *operatorActions) ScaleTidbCluster(info *TidbClusterInfo) error   { return nil }
-func (oa *operatorActions) UpgradeTidbCluster(info *TidbClusterInfo) error { return nil }
-func (oa *operatorActions) DeployMonitor(info *TidbClusterInfo) error      { return nil }
-func (oa *operatorActions) CleanMonitor(info *TidbClusterInfo) error       { return nil }
+func chartPath(name string, tag string) string {
+	return "/charts/" + tag + "/" + name
+}
+
+func (oa *operatorActions) ScaleTidbCluster(info *TidbClusterInfo) error {
```

Comment:
change to `ScaleTidbClusterTo(info *TidbClusterInfo,pdReplicas int,tikvReplicas int,tidbReplicas int)` is more reasonable